src/app/backend/search/search.go

```diff
@@ -43,16 +43,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-// SearchResult is a list of resources matching search criteria found in whole cluster.
-type SearchResult struct {
-
-	// Cluster.
-	NamespaceList        namespace.NamespaceList               `json:"namespaceList"`
-	NodeList             node.NodeList                         `json:"nodeList"`
-	PersistentVolumeList persistentvolume.PersistentVolumeList `json:"persistentVolumeList"`
-	RoleList             rbacroles.RbacRoleList                `json:"roleList"`
-	StorageClassList     storageclass.StorageClassList         `json:"storageClassList"`
-
+// AllObjectsInNamespace is a struct containing all of the objects in a given namespace
+type AllObjectsInNamespace struct {
```

Comment:
@maciaszczykm what do you think of `ResourcesInNamespace`? Since namespace is also a kind of object I think we should be as clear as possible that these are subordinate to a namespace.